app/Filament/Resources/EmulatorResource.php

```diff
@@ -73,34 +73,46 @@ public static function infolist(Infolist $infolist): Infolist
 
                                     Infolists\Components\TextEntry::make('original_name')
                                         ->label('Original name')
-                                        ->helperText('Original name of emulator.'),
+                                        ->placeholder('none')
+                                        ->helperText("If this emulator is forked from another emulator, put the original emulator's name here. eg: 'Snes9x' is the original name for 'RASnes9x'."),
 
                                     Infolists\Components\TextEntry::make('description')
                                         ->label('Description')
-                                        ->helperText('Additional text to display on the download page.'),
+                                        ->placeholder('none')
+                                        ->helperText("This text used to be displayed on the Downloads page, but isn't anymore. You can add notes about the emulator here."),
 
                                     Infolists\Components\TextEntry::make('documentation_url')
-                                        ->label('Documentation link')
-                                        ->helperText('Link to emulator documentation.'),
+                                        ->label('Documentation URL')
+                                        ->url(fn ($record) => $record->documentation_url)
+                                        ->placeholder('none')
+                                        ->helperText('This appears on the Downloads page.'),
 
                                     Infolists\Components\TextEntry::make('website_url')
-                                        ->label('Website link')
-                                        ->helperText('Link to emulator homepage.'),
+                                        ->label('Website URL')
+                                        ->url(fn ($record) => $record->website_url)
+                                        ->placeholder('none')
+                                        ->helperText('This appears on the Downloads page.'),
 
                                     Infolists\Components\TextEntry::make('source_url')
-                                        ->label('Source code link')
-                                        ->helperText('Link to emulator source code.'),
+                                        ->label('Source code URL')
+                                        ->url(fn ($record) => $record->source_url)
+                                        ->placeholder('none')
+                                        ->helperText('This appears on the Downloads page.'),
 
                                     Infolists\Components\TextEntry::make('download_url')
-                                        ->label('Download link')
+                                        ->label('Download URL')
+                                        ->url(fn ($record) => $record->download_url)
```

Comment:
Our forked emulators use relative URLs, which this isn't happy with.

<img width="388" height="134" alt="Image" src="https://github.com/user-attachments/assets/ae18aac5-ca18-400b-9d7d-f32d6fb12669" />

I think it would be safe to change these to full URLs if we ever did need to edit the associated records.